packages/components/src/internal/test/enzymeTestHelpers.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, ReactNode } from 'react';
```

Comment:
Probably on your list to circle back around to this file and clean up imports / unused functions.